atoma-proxy/src/server/middleware.rs

```diff
@@ -596,7 +603,7 @@ pub mod auth {
         headers: &HeaderMap,
         body_json: &Value,
         endpoint: &str,
-    ) -> Result<(Option<Stack>, u64, String, i64)> {
+    ) -> Result<(Option<Stack>, u64, u64, String, i64)> {
```

Comment:
this tuple is getting out of hand, change it to struct

Reply:
Addressed, thanks for pointing it out